app/middleware/jwt/user_authenticator.rb

```diff
@@ -4,7 +4,10 @@ def validate(request_headers)
       @request_headers = request_headers
 
       payload, _header = Jwt::TokenDecryptor.decrypt(token)
-      User.find(payload['user_id'])
+      user = User.find(payload['user_id'])
+      return nil if user.archived?
```

Comment:
you have added [scope](https://github.com/vancityjun/focus-backend/pull/4/files#diff-89dbb72e13b9201b4f706179cb9f9b95b32eb39166eaad7a07730fa1568026f0R8) on `archivable.rb`. could we use it so that `user` returns `nil` if it's archived?

Reply:
Thanks for pointing this out. I will update it.